### PR TITLE
migrate the notes app to owncloud 8

### PR DESCRIPTION
**Pull request by Raydiation**
Requires:

* [x] https://github.com/owncloud/core/pull/13667
* [x] https://github.com/owncloud/core/pull/13676/files
* [x] Fix unit tests

### CODE REVIEW

File: service/notesservice.php [hunk 1]
```diff
@@ -140,38 +110,102 @@ public function update($id, $content){
...
+		return $file[0];
+	}
+
+
+	/**
+	 * @param string $userId the user id
+	 * @return Folder
+	 */
+	private function getFolderForUser ($userId) {
+		$path = '/' . $userId . '/files/notes';
```
Comment (icewind1991) [inline service/notesservice.php]: This was `'files/Notes'` before

Comment (Raydiation) [inline service/notesservice.php]: Yes, but the folder that went in was already in the /$userId/

Comment (icewind1991) [inline service/notesservice.php]: I mean the folder used to be with an upper case 'N'

**Comment by Raydiation** [comment]
@LukasReschke unit tests + migration done :)

File: service/notesservice.php [hunk 2]
```diff
@@ -140,38 +110,98 @@ public function update($id, $content){
 		$title = str_replace(array('/', '\\'), '',  $title);
```
Comment (LukasReschke) [inline service/notesservice.php]: I wonder if we can get rid of this by using a view on `Notes`

Comment (Raydiation) [inline service/notesservice.php]: Should already be a view I think

**Pull request merged**